cmd/notary/keys.go

```diff
@@ -424,6 +425,19 @@ func (k *keyCommander) keysRotate(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+
+	if rotateKeyRole == data.CanonicalRootRole {
+		cmd.Println("Warning: you are about to rotate your root key.\n" +
+			"You will still need your old key to sign future root changes\n" +
+			"so that clients who may not be behind can update.\n" +
```

Comment:
Good point, this is pretty unclear.  I was trying to imply that you'd need it forever, not just for this rotation - I'll rephrase.